CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ add_library(expected INTERFACE)
 target_include_directories(expected
   INTERFACE
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
-    $<INSTALL_INTERFACE:include>)
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
```

Comment:
This hunk is unrelated to Catch2, no?

Reply:
Yes. It fixes installation on some distributions and configurations where `$prefix/include` is not `$prefix/include`.